circle.yml

```diff
@@ -18,6 +18,16 @@ dependencies:
 test:
   override:
     - mvn -B test
+    # Run Docker-in-Docker tests
+    # Yes, we use selenium image, but we use it anyway in BrowserContainer's tests
+    - >
+      docker -H unix:///var/run/docker.sock run --rm \
```

Comment:
had to duplicate. I can extract it to .sh file to share, but we don't use CircleCI anymore, right? Probably the config should be removed then